tools/ya-test-runner/ya_test_runner/stage/execution.py

```
@@ -0,0 +1,119 @@
+		)
+	finally:
+		if not success:
+			ctx.failed.append(case.description.name)
+		latch.decrement()
+
+
+async def _run_cases(
+	ctx: _ExecutionContext, cases: list[ya_test_runner.test.Case], latch: _CountDownLatch
+):
+	for case in cases:
+		asyncio.create_task(_run_case(ctx, case, latch))
```

Ruff (0.14.7)

Store a reference to the return value of `asyncio.create_task`

(RUF006)